vm-presets.md

```diff
@@ -0,0 +1,237 @@
+Virtual Machine Presets
+=============================
+
+`VirtualMachinePresets` are an extension to general `VirtualMachine`
+Configuration based heavily on `PodPresets` from Kubernetes. When a
+`VirtualMachine` is created, any applicable `VirtualMachinePresets`
+will be applied to the existing spec for the `VirtualMachine`. This allows
+for re-use of common settings that should apply to multiple `VirtualMachines`.
+
+Usage
+------------------------
+
+KubeVirt uses Kubernetes `Labels` to determine which `VirtualMachinePresets`
+apply to any given `VirtualMachine`, similarly to how `PodPresets` work in
+Kubernetes. The `VirtualMachine` is marked with an Annotation upon successful
+completion.
+
+```yaml
```

Comment:
Please add some context to this example.

Is this before or after a preset?

If after, then pls add the expected annotation.

Reply:
An example covering this exact same interaction occurs later in the document, I think I'll remove it here because I agree that it conflicts with the text.